packages/api/src/localization/en-US.json

```diff
@@ -123,6 +123,9 @@
   "TEXT_INPUT_SEND_BUTTON_ALT": "Send",
   "TEXT_INPUT_SPEAK_BUTTON_ALT": "Speak",
   "TEXT_INPUT_UPLOAD_BUTTON_ALT": "Upload file",
+  "TEXT_INPUT_DIALPAD_BUTTON_ALT": "Dialpad",
+  "TEXT_INPUT_DROPZONE": "Drop files",
+  "TEXT_INPUT_ATTACHMENTS": "attachments",
```

Comment:
Plural need to use another form. Zero, one, few, many, others.

You can see the `TOAST_ACCORDION_*` below.

Reply:
Pending design decision on this

Reply:
Strings added